### DISCUSSION TITLE
Add an option to make tapping weekly widget open the main screen

### DISCUSSION THREAD

**Discussion by helgridly** [breezy-weather/breezy-weather#1026] [Ideas]
### Describe your idea

I have the 5-day forecast widget on my desktop. When I tap it, it opens the forecast for the day I tapped. I'd like it to open the main screen with the cards instead. Would it be possible to make this an option in the settings?

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.2 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
It opens the day details when you tap on a specific day. If you tap anywhere else, it opens the main screen.

**Reply by helgridly** [reply]
Yeah, I just have fat thumbs and don't think about where I'm tapping.
